### PR TITLE
Add new sample code to developers.optimizely.com

### PR DESCRIPTION
**Pull request by jgaul**
Also clarify the difference between the old and new results API examples.

@aliabbasrizvi @thomascirca @tscanlin @jprovine

### CODE REVIEW

File: src/content/samples/results/0-intro.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
 ---
 template: inline
-title: Experiment results
+title: Experiment results (pre-Stats-Engine)
```
Comment (aliabbasrizvi) [inline src/content/samples/results/0-intro.md]: Do we want to mention experiments started before January 21, 2015?

Comment (jgaul) [inline src/content/samples/results/0-intro.md]: i figured i'd rather avoid more info duplication between this and the endpoint reference

**Comment by tscanlin** [comment]
LGTM

**Comment by jprovine** [comment]
LGTM, make sure any changes to the code samples go through the SA team
(Brad Taylor or Spencer Smitherman)

On Mon, Jun 15, 2015 at 6:04 PM, Jon Gaulding <notifications@github.com>
wrote:

> Also clarify the difference between the old and new results API examples.
>
> @aliabbasrizvi <https://github.com/aliabbasrizvi> @thomascirca
> <https://github.com/thomascirca> @tscanlin <https://github.com/tscanlin>
> @jprovine <https://github.com/jprovine>
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/optimizely/developers.optimizely.com/pull/78
> Commit Summary
>
>    - Add new sample code to developers.optimizely.com, and clarify the
>    difference between the old and new results API examples.
>
> File Changes
>
>    - *M* src/content/samples/index.concat
>    <https://github.com/optimizely/developers.optimizely.com/pull/78/files#diff-0>
>    (2)
>    - *M* src/content/samples/results/0-intro.md
>    <https://github.com/optimizely/developers.optimizely.com/pull/78/files#diff-1>
>    (7)
>    - *A* src/content/samples/stats/0-intro.md
>    <https://github.com/optimizely/developers.optimizely.com/pull/78/files#diff-2>
>    (13)
>    - *A* src/content/samples/user-lists/0-intro.md
>    <https://github.com/optimizely/developers.optimizely.com/pull/78/files#diff-3>
>    (15)
>
> Patch Links:
>
>    - https://github.com/optimizely/developers.optimizely.com/pull/78.patch
>    - https://github.com/optimizely/developers.optimizely.com/pull/78.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/optimizely/developers.optimizely.com/pull/78>.
>



-- 
*John Provine*
Product Manager, Optimizely
john@optimizely.com | 917.756.3717

**Comment by jgaul** [comment]
Sent a heads up to the SAs and Brad an Spencer specifically.

**Pull request merged**